packages/babel-preset-typescript/README.md

```diff
@@ -0,0 +1,54 @@
+# babel-preset-typescript
+
+> Babel preset for TypeScript.
+
+This preset includes the following plugins:
+
+- [transform-typescript](https://babeljs.io/docs/plugins/transform-typescript/)
+- [syntax-object-rest-spread](https://babeljs.io/docs/plugins/syntax-object-rest-spread/)
```

Comment:
Ok I thought that either babel-plugin-syntax-typescript would just include it there rather than putting it in the preset https://github.com/babel/babel/pull/5899/files#diff-36ceea99653860f193c748d23736fe0aR4

or that babylon would have a similar || check for object spread https://github.com/babel/babylon/blob/c88af90c0a88ec756ad7c27f0a1c6f92bffbcd24/src/parser/statement.js#L1148 - just didn't seem consistent for both class properties and object spread.

Can handle in a followup PR though